src/core/checkedint.d

```diff
@@ -492,8 +494,8 @@ unittest
 pragma(inline, true)
 ulong mulu(ulong x, ulong y, ref bool overflow)
 {
-    ulong r = x * y;
-    if (x && (r / x) != y)
+    const r = x * y;
+    if (x && r / x != y)
```

Comment:
I don't understand. If we can write `a + b == x` why can't we delete parenthesis in `(a + b) == x`?